tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -215,7 +213,6 @@ def test_eof_validity(
                     max_stack_height=2,
                 ),
             ],
-            validity_error=EOFException.TRUNCATED_INSTRUCTION,
```

Comment:
the `eof_test` invocation already names the expected error, why the need to duplicate here?

Reply:
This removes the duplication.

Reply:
omg I'm blind :facepalm: